reactive/openvim.py

```diff
@@ -4,10 +4,26 @@
 from charmhelpers.core.unitdata import kv
 from charmhelpers.core.host import mkdir, symlink, chownr, add_user_to_group
 from charmhelpers.fetch.archiveurl import ArchiveUrlFetchHandler
+from charmhelpers.contrib.unison import ensure_user
 
-def add_user_to_libvirt_group():
-    status_set("maintenance", "adding user to libvirtd group")
-    add_user_to_group("ubuntu", "libvirtd")
+def create_openvim_user():
+    status_set("maintenance", "creating openvim user")
+    ensure_user('openvim')
+
+def group_openvim_user():
+    status_set("maintenance", "adding openvim user to groups")
+    add_user_to_group('openvim', 'libvirtd')
+    add_user_to_group('openvim', 'sudo')
```

Comment:
Yeah, see for example https://github.com/nfvlabs/openmano/blob/master/openvim/utils/RADclass.py#L870